operator/pkg/ratelimit/jitter.go

```diff
@@ -5,17 +5,31 @@ import (
 	"time"
 )
 
-var randomSeed = rand.New(rand.NewSource(time.Now().UnixNano()))
+//nolint:gosec,gochecknoglobals
+var RandomSeed = rand.New(rand.NewSource(time.Now().UnixNano()))
+
+const (
+	tenPercentMultiplier = 0.1
+)
 
 // RandomizeByTenPercent returns the value that is randomly changed by at most +/- 10% from the input.
 // May return the same value.
 func RandomizeByTenPercent(val int) int {
-	fv := float64(val)
+	floatValue := float64(val)
+
+	tenPercentOfVal := tenPercentMultiplier * floatValue
 
-	var tenPercentOfVal float64 = 0.1 * fv
+	randomFactor := 1 - RandomSeed.Float64() // random number in range [-1 ... 0 ... +1)
+	delta := randomFactor * tenPercentOfVal  // +- 10%
 
-	randomFactor := 1 - randomSeed.Float64()*2 // random number in range [-1 ... 0 ... +1)
-	delta := randomFactor * tenPercentOfVal    // +- 10%
+	return int(floatValue + delta)
 
-	return int(fv + delta)
+	// TODO: replace with crypto/rand
+	// timeVal := big.NewInt(time.Now().UnixNano())
+	// randomSeed, err := rand.Int(rand.Reader, timeVal)
+	// if err != nil {
+	//	return 0, err
+	//}
+	//
+	// return randomSeed.Int64(), nil
```

Comment:
```suggestion

```

Reply:
I added this purposely in order to replace it in another PR